app/views/content_items/_detailed_guide_b.html.erb

```diff
@@ -0,0 +1,54 @@
+<%= content_for :title, @content_item.page_title %>
```

Comment:
@fofr I'm not sure I understand what you're proposing. Surely the only way to be able to have arbitrarily different HTML views is to have two different templates (or variants)?